### DISCUSSION TITLE
Using detectron 2 to show the performance metrics

### DISCUSSION THREAD

**Discussion by ck12780** [facebookresearch/detectron2#5232] [Q&A]
Hi, i am a newbie for coding. I am using detectron2 and pytorch for doing a custom object detection.
![Screenshot 2024-03-08 142051](https://github.com/facebookresearch/detectron2/assets/155506569/57f789a8-8cc4-4800-9bef-98f2ee00a353)
So, above image is the evaluation result for my training. How to code for the accuracy, precision, F1 score and confusion matric.
The training code I take from 
https://github.com/ahmad12hamdan99/Rovers_vs_Cars/blob/main/detectron2.ipynb
